scripts/seed.py

```diff
@@ -0,0 +1,210 @@
+# Command to run : python manage.py shell --settings=settings.dev  < scripts/seed.py
+
+from django.conf import settings
+from django.core.files.uploadedfile import SimpleUploadedFile
+from django.contrib.auth.models import User
+from django.utils import timezone
+
+from allauth.account.models import EmailAddress
+
+from datetime import timedelta
+from faker import Factory
+
+import os
+
+from challenges.models import Challenge, ChallengePhase, DatasetSplit, Leaderboard, ChallengePhaseSplit
+from hosts.models import ChallengeHostTeam, ChallengeHost
+from participants.models import Participant, ParticipantTeam
+
+
+def create_admin_user():
+    username = "admin"
+    password = "password"
+    email = "admin@gmail.com"
+    user = User.objects.create_user(
+        email=email,
+        username=username,
+        password=password,
+        is_staff=True,
+        is_superuser=True,
+    )
+    EmailAddress.objects.create(user=user, email=email, verified=True, primary=True)
+    print "Super user created with \n username: %s \n password: %s" % (username, password)
+
+
+def create_user(username=""):
+    Factory.create()
+    password = "password"
+    email = "%s@gmail.com" % (username)
+    user = User.objects.create_user(
+        email=email,
+        username=username,
+        password=password,
+    )
+    EmailAddress.objects.create(user=user, email=email, verified=True, primary=True)
+    print "User created with \n username: %s \n email: %s \n password: %s" % (username, email, password)
+    return user
+
+
+def create_challenge_host_team(user):
+    fake = Factory.create()
+    team_name = "%s Host Team" % (fake.city())
+    team = ChallengeHostTeam.objects.create(
+        team_name=team_name,
+        created_by=user,
+    )
+    print "Challenge Host Team created with \n team_name: %s \n created_by: %s" % (team_name, user.username)
+    ChallengeHost.objects.create(user=user, team_name=team, status="Self", permissions="Admin")
```

Comment:
I think we can use ChallengeHost.SELF and ChallengeHost.ADMin for `Self` and `Admin`. That actually makes it more accurate. Can you also make sure that this is followed everywhere where in some string value related to model is being used. Also update us if any model is missing such value.